lib/tests/queryManager.test.js

```diff
@@ -0,0 +1,41 @@
+import queryManager from "../integration/queryManager";
+import dbConnector from "../integration/dbConnector";
+
+jest.mock("../integration/dbConnector", () => {
```

Comment:
If you're mocking dbConnector you're not testing dbConnector. You'll need to write tests for dbConnector but good use of a mock for queryManager. Although you can see how this is a lot simpler if you pass in the things the function needs rather than having it as a side effect